src/system/file/chown.rs

```diff
@@ -0,0 +1,26 @@
+use std::{fs::File, io, os::fd::AsRawFd};
+
+use crate::{
+    cutils::cerr,
+    system::interface::{GroupId, UserId},
+};
+
+mod sealed {
+    use std::fs::File;
+
+    pub(crate) trait Sealed {}
```

Comment:
I don't see much benefit to sealing the `Chown` trait because it's not an API that's exposed to the dependents of this crate.